package.json

```diff
@@ -10,16 +10,18 @@
   "license": "MIT",
   "repository": "jsdom/whatwg-url",
   "dependencies": {
+    "stable": "^0.1.6",
```

Comment:
There's ["lodash.sortby"](https://www.npmjs.com/package/lodash.sortby) so I guess I can use that, though its operation looks more complicated than "stable"'s that I would actually prefer using "stable". Either way I'll change it to lodash for the time being, and I've filed https://github.com/Two-Screen/stable/issues/6.